src/js/plugin-bridge/__tests__/PluginSDK-test.js

```diff
@@ -48,7 +48,7 @@ describe("PluginSDK", function() {
         expect(deepEqual(state.fakePlugin2, { foo: "bar" })).toEqual(true);
       });
 
-      it("should throw error if reducer is not a function", function() {
+      it("throws error if reducer is not a function", function() {
```

Comment:
*throws an error

Reply:
Fixing the grammar is not in scope of this PR.